### PR TITLE
Change copyright

### PR DESCRIPTION
**Pull request by dneto0**
Three commits here:
1. Fix up Google copyright lines so they all say 'Google Inc.' as the author. (Useful for the checker)
2. Add a checker / updater script.
3. Apply the script, and manually update the README and external/CMakeLists.txt, and change LICENSE to be the plain Apache 2.0 license.

### CODE REVIEW

File: tools/lesspipe/spirv-lesspipe.sh [hunk 1]
```diff
@@ -2,29 +2,17 @@
...
-#    https://www.khronos.org/registry/
-#
-# THE MATERIALS ARE PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
-# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
-# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
-# IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
-# CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
-# TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
-# MATERIALS OR THE USE OR OTHER DEALINGS IN THE MATERIALS.
+// Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (antiagainst) [inline tools/lesspipe/spirv-lesspipe.sh]: comment style ;)

File: utils/check_copyright.py [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+                description=inspect.getdoc(sys.modules[__name__]))
+        self.add_argument('--update', dest='author', action='store',
+                          help='For files missing a copyright notice, insert '
+                               'one for the given author, and add a license '
+                               'notice.  The author must be in the AUTHORS '
+                               'list in the script.')
+
+
+def main():
+    glob_comment_pairs = [('*.h', '//'), ('*.hpp', '//'), ('*.sh', '//'),
```
Comment (antiagainst) [inline utils/check_copyright.py]: `shell` should use `#`.

File: utils/check_copyright.py [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+AUTHORS = ['The Khronos Group Inc.',
+           'Google Inc.']
+CURRENT_YEAR='2016'
+
+YEARS = '(2014-2016|2015-2016|2016)'
+COPYRIGHT_RE = re.compile(
+        'Copyright \(c\) {} ({})'.format(YEARS, '|'.join(AUTHORS)))
+
+MIT_BEGIN_RE = re.compile('Permission is hereby granted, '
+                                 'free of charge, to any person obtaining a')
```
Comment (antiagainst) [inline utils/check_copyright.py]: format

File: utils/check_copyright.py [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+def main():
+    glob_comment_pairs = [('*.h', '//'), ('*.hpp', '//'), ('*.sh', '//'),
+                          ('*.py', '#'), ('*.cpp', '//'),
+                          ('CMakeLists.txt', '#')]
+    argparser = ArgParser()
+    args = argparser.parse_args()
+
+    if args.author:
+        if args.author not in AUTHORS:
+            print('error: --author must be in the AUTHORS list in '
```
Comment (antiagainst) [inline utils/check_copyright.py]: This actually suggests we are using `--author` as the command line option, while the above listed as `--update`?

File: utils/check_copyright.py [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+                  'check_copyright.py: {}'.format(AUTHORS))
+            sys.exit(1)
+        for pair in glob_comment_pairs:
+            insert_copyright(args.author, *pair)
+        sys.exit(0)
+    else:
+        count = 0
+        for pair in glob_comment_pairs:
+            count += alert_if_no_copyright(*pair)
```
Comment (antiagainst) [inline utils/check_copyright.py]:
```python
count = sum([alert_if_no_copyright(*p) for p in glob_comment_pairs])
```

File: utils/check_copyright.py [hunk 6]
```diff
@@ -0,0 +1,219 @@
...
+
+def comment(text, prefix):
+    """Returns commented-out text.
+
+    Each line of text will be prefixed by prefix and a space character.  Any
+    trailing whitespace will be trimmed.
+    """
+    accum = []
+    for line in text.split('\n'):
+        accum.append((prefix + ' ' + line).rstrip())
```
Comment (antiagainst) [inline utils/check_copyright.py]:
```python
accum = ['{} {}'.format(prefix, line).rstrip() for line in text.split('\n')]
```

**Comment by dneto0** [comment]
All feedback addressed.

**Comment by antiagainst** [comment]
+2

**Pull request merged**